test/charms/test_main/src/charm.py

```diff
@@ -132,37 +133,45 @@ def on_db_relation_joined(self, event):
         self._write_state()
 
     def on_mon_relation_changed(self, event):
-        assert event.app is not None, 'application name cannot be None for a relation-changed event'
+        assert event.app is not None, \
+            'application name cannot be None for a relation-changed event'
         if os.environ.get('JUJU_REMOTE_UNIT'):
-            assert event.unit is not None, 'a unit name cannot be None for a relation-changed event associated with a remote unit'
+            assert event.unit is not None, \
+                'a unit name cannot be None for a relation-changed event' \
+                ' associated with a remote unit'
```

Comment:
(from PEP8)

 Try to avoid as much as possible the use of backlash to not-end-of-line. One alternative here is to do:
```
            assert event.unit is not None, (
                'a unit name cannot be None for a relation-changed event' 
                ' associated with a remote unit')
```